extensions/wikia/Search/classes/IndexService/DefaultContent.php

```diff
@@ -269,22 +303,34 @@ protected function extractInfoboxes( simple_html_dom $dom ) {
 						$headerCells = $row->find( 'th' );
 						$infoBoxCellCount = count( $infoboxCells );
 						$headerCellCount = count( $headerCells );
-						if ( $infoBoxCellCount == 2  && $headerCellCount == 0 ) {
-							$result['infoboxes_txt'][] = "infobox_{$counter} | " . preg_replace( '/\s+/', ' ', $infoboxCells[0]->plaintext . ' | ' . $infoboxCells[1]->plaintext  );
+						if ( $infoBoxCellCount == 2 && $headerCellCount == 0 ) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Refactor this code to not nest more than 4 "if", "for", "while", "switch" and "try" statements. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS134)